src/dkim/parse.rs

```diff
@@ -651,26 +632,7 @@ mod test {
                     "/RtdC2UzJ1lWT947qR+Rcac2gbto/NMqJ0fzfVjH4OuKhi",
                     "tdY9tf6mcwGjaNBcWToIMmPSPDdQPNUYckcQ2QIDAQAB",
                 ),
-                DomainKey {
-                    v: Version::V1,
-                    p: PublicKey::Rsa(
-                        RsaPublicKey::from_public_key_der(
-                            &base64_decode(
-                                concat!(
-                                    "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQ",
-                                    "KBgQDwIRP/UC3SBsEmGqZ9ZJW3/DkMoGeLnQg1fWn7/zYt",
-                                    "IxN2SnFCjxOCKG9v3b4jYfcTNh5ijSsq631uBItLa7od+v",
-                                    "/RtdC2UzJ1lWT947qR+Rcac2gbto/NMqJ0fzfVjH4OuKhi",
-                                    "tdY9tf6mcwGjaNBcWToIMmPSPDdQPNUYckcQ2QIDAQAB",
-                                )
-                                .as_bytes(),
-                            )
-                            .unwrap(),
-                        )
-                        .unwrap(),
-                    ),
-                    f: 0,
-                },
+                0,
```

Comment:
We can't easily compare the trait object key types, but we're testing that via actual signature verification anyway in other tests, so it seemed fine to just exercise the flag parsing here.